addon/lib/field.js

```diff
@@ -187,6 +187,39 @@ export default Base.extend({
     return !this.answer || this.answer.isNew;
   }),
 
+  /**
+   * Whether the field has the defined default answer of the question as value.
+   *
+   * @property {Boolean} isDefault
+   * @accessor
+   */
+  isDefault: computed(
+    "value.@each.flatAnswerMap",
+    "question.{isTable,defaultValue}",
+    function () {
+      if (!this.value || !this.question.defaultValue) return false;
+
+      const value = this.question.isTable
+        ? this.value.map((doc) => doc.flatAnswerMap)
+        : this.value;
+
+      return (
+        JSON.stringify(value) === JSON.stringify(this.question.defaultValue)
```

Comment:
This worried me a bit as the order of object properties is not guaranteed. But from what I've learned they should respect the order they were set in.

So, my question becomes: Is the order of `this.fields` dependent on configuration or the order in which they were filled in by the user? If it is by configuration, this comparison should not be a problem.

Reply:
Very good point. The order of the fields is based on configuration, however the order of the default answer is not.. I'll have to find another way of comparing those. Do you have experience with https://lodash.com/docs/4.17.15#isEqual ?

Reply:
I remember using it a lot with React to check if the properties had changed. *I don't remember having any issues with it.* But, that was a long time ago and maybe I suppressed a bad memory including isEqual :smile: 

But yeah. I agree that this sounds like a good idea!

Reply:
Tested it and it works. I also added a test, will push soon. Good catch!

Reply:
:+1: